Formula/llvm.rb

```diff
@@ -114,21 +114,27 @@ def install
       -DCLANG_VENDOR_UTI=org.#{tap.user.downcase}.clang
     ]
 
+    sdk = MacOS.sdk_path_if_needed
     if MacOS.version >= :catalina
-      args << "-DFFI_INCLUDE_DIR=#{MacOS.sdk_path}/usr/include/ffi"
-      args << "-DFFI_LIBRARY_DIR=#{MacOS.sdk_path}/usr/lib"
+      args << "-DFFI_INCLUDE_DIR=#{sdk}/usr/include/ffi"
+      args << "-DFFI_LIBRARY_DIR=#{sdk}/usr/lib"
     else
       args << "-DFFI_INCLUDE_DIR=#{Formula["libffi"].opt_include}"
       args << "-DFFI_LIBRARY_DIR=#{Formula["libffi"].opt_lib}"
     end
 
-    sdk = MacOS.sdk_path_if_needed
+    # gcc-5 fails at building compiler-rt. Enable PGO
+    # build on Linux when we switch to Ubuntu 18.04.
+    pgo_build = false
     on_macos do
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
       args << "-DLLVM_CREATE_XCODE_TOOLCHAIN=#{MacOS::Xcode.installed? ? "ON" : "OFF"}"
```

Comment:
This will always be `ON` on CI I think

Reply:
Do you mean `LLVM_CREATE_XCODE_TOOLCHAIN`? That's true. I'll move it out of the `on_macos` block so we don't need to declare it in two places.